### PR TITLE
Filter itest

### PR DESCRIPTION
**Pull request by jsanda**
Adding a JSON integration test for filters. This helps get me more familiar with filter-related code so I can put together some prototypes for review/discussion.

### CODE REVIEW

File: src/test/resources/org/usergrid/vx/experimental/filter_beers.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            "type": "value",
+            "clazz": "UTF-8"
+        }
+    },
+    {
+        "type": "CREATEFILTER",
+        "op": {
+            "name": "stouts",
+            "spec": "groovy",
+            "value": "class Stout implements org.usergrid.vx.experimental.Filter {\n  Map filter(Map row)  {\n  if (row[\"value\"] == \"Breakfast Stout\") {\n  return row\n} else {\n  return null\n} } }"
```
Comment (edwardcapriolo) [inline src/test/resources/org/usergrid/vx/experimental/filter_beers.json:37]: I'd drink to that.

**Comment by zznate** [comment]
More merge sadness :( 

Otherwise, lgtm.

**Comment by jsanda** [comment]
Want me to handle resolving the conflicts?

**Comment by edwardcapriolo** [comment]
Yes go for it. If not I can look at it in a couple of hours.

**Comment by jsanda** [comment]
The history on my local branch looked all out of whack. I cherry picked the commit, and that avoided merge conflicts. The commit has been pushed.

**Pull request closed without merging**